config/crd/policy.networking.k8s.io_adminnetworkpolicies.yaml

```diff
@@ -24,6 +24,18 @@ spec:
     - jsonPath: .metadata.creationTimestamp
       name: Age
       type: date
+    - jsonPath: .status.conditions[*].type
```

Comment:
what happens when we have multiple conditions on the status?

Reply:
@danwinship : So I guess its like we said, only the first one get's printed :/ 
```
Status:
  Conditions:
    Last Transition Time:  2023-06-07T06:48:10Z
    Message:               Setting up OVN DB plumbing was successful
    Reason:                SetupSucceeded
    Status:                True
    Type:                  AdminNetworkPolicyReady
    Last Transition Time:  2023-06-07T06:48:10Z
    Message:               Setting up OVN DB plumbing was successful
    Reason:                SetupSucceeded
    Status:                True
    Type:                  SURYA
Events:                    <none>
```
versus
```
$ oc get anp
NAME           PRIORITY   AGE   CONDITION                 STATUS   REASON
pass-example   10         3s    AdminNetworkPolicyReady   True     SetupSucceeded
```

Reply:
the other way round is to do conditional things like:
```
- additionalPrinterColumns:
        - jsonPath: .status.history[?(@.state=="Completed")].version
          name: Version
          type: string
        - jsonPath: .status.conditions[?(@.type=="Available")].status
          name: Available
          type: string
        - jsonPath: .status.conditions[?(@.type=="Progressing")].status
          name: Progressing
          type: string
        - jsonPath: .status.conditions[?(@.type=="Progressing")].lastTransitionTime
          name: Since
          type: date
        - jsonPath: .status.conditions[?(@.type=="Progressing")].message
          name: Status
          type: string
```
but since we don't standardize the types here... we can't do that.. So I guess printing something better than nothing?